WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1308,226 +1346,230 @@ def removeJobFromDB( self, jobIDs ):
 
     if not result['OK']:
       return result
 
-    priority = classAdJob.getAttributeInt( 'Priority' )
+    priority = classAdJob.getAttributeInt('Priority')
     if priority is None:
       priority = 0
-    jobAttrNames.append( 'UserPriority' )
-    jobAttrValues.append( priority )
+    jobAttrNames.append('UserPriority')
+    jobAttrValues.append(priority)
 
-    siteList = classAdJob.getListFromExpression( 'Site' )
+    siteList = classAdJob.getListFromExpression('Site')
     if not siteList:
       site = 'ANY'
-    elif len( siteList ) > 1:
+    elif len(siteList) > 1:
       site = "Multiple"
     else:
       site = siteList[0]
 
-    jobAttrNames.append( 'Site' )
-    jobAttrValues.append( site )
+    jobAttrNames.append('Site')
+    jobAttrValues.append(site)
 
-    jobAttrNames.append( 'Status' )
-    jobAttrValues.append( 'Received' )
+    jobAttrNames.append('Status')
+    jobAttrValues.append('Received')
 
-    jobAttrNames.append( 'MinorStatus' )
-    jobAttrValues.append( 'Job Rescheduled' )
+    jobAttrNames.append('MinorStatus')
+    jobAttrValues.append('Job Rescheduled')
 
-    jobAttrNames.append( 'ApplicationStatus' )
-    jobAttrValues.append( 'Unknown' )
+    jobAttrNames.append('ApplicationStatus')
+    jobAttrValues.append('Unknown')
 
-    jobAttrNames.append( 'ApplicationNumStatus' )
-    jobAttrValues.append( 0 )
+    jobAttrNames.append('ApplicationNumStatus')
+    jobAttrValues.append(0)
 
-    jobAttrNames.append( 'LastUpdateTime' )
-    jobAttrValues.append( Time.toString() )
+    jobAttrNames.append('LastUpdateTime')
+    jobAttrValues.append(Time.toString())
 
-    jobAttrNames.append( 'RescheduleTime' )
-    jobAttrValues.append( Time.toString() )
+    jobAttrNames.append('RescheduleTime')
+    jobAttrValues.append(Time.toString())
 
     reqJDL = classAdReq.asJDL()
-    classAdJob.insertAttributeInt( 'JobRequirements', reqJDL )
+    classAdJob.insertAttributeInt('JobRequirements', reqJDL)
 
     jobJDL = classAdJob.asJDL()
 
     # Replace the JobID placeholder if any
-    if jobJDL.find( '%j' ) != -1:
-      jobJDL = jobJDL.replace( '%j', str( jobID ) )
+    if jobJDL.find('%j') != -1:
+      jobJDL = jobJDL.replace('%j', str(jobID))
 
-    result = self.setJobJDL( jobID, jobJDL )
+    result = self.setJobJDL(jobID, jobJDL)
     if not result['OK']:
       return result
 
-    result = self.__setInitialJobParameters( classAdJob, jobID )
+    result = self.__setInitialJobParameters(classAdJob, jobID)
     if not result['OK']:
       return result
 
-    result = self.setJobAttributes( jobID, jobAttrNames, jobAttrValues )
+    result = self.setJobAttributes(jobID, jobAttrNames, jobAttrValues)
     if not result['OK']:
       return result
 
-    retVal['InputData'] = classAdJob.lookupAttribute( "InputData" )
+    result = self.setJobStatus(jobID, status='Received', minor='Job Rescheduled', application='Unknown')
+    if not result['OK']:
+      return result
```

Comment:
Since you call `setJobStatus`, you should at this point remove lines 1495 -> 1505.